.github/workflows/scripts/e2e-verify.common.sh

```diff
@@ -107,23 +115,38 @@ verify_provenance_authenticity() {
         return 0
     fi
 
+    # Assemble artifact args: if $BINARY is defined, then this is an artifact, otherwise an image.
+    if [[ -n $BINARY ]]; then
```

Comment:
nit: You might try using `"$(echo "$THIS_FILE" | cut -d '.' -f2)" == "container"` to check if this is a container e2e test or not.

Reply:
even better! I'll add a TODO with the GCB test issue that it would need to check gcb/container.